### ISSUE TITLE
Enumerator error - Program crashes after printing potential infected file path.

### ISSUE THREAD

**Issue by Cian-R** [E-B3rry/yoursearchbar.me-removal#1]
Program sucessfully downloaded missing .dll files.
It then spent a few seconds searching for infected files, appears to have found one which it prints to console, and then immediately crashes.

Probably a minor issue, as this file seems like the sole likely culprit.

Output below (with screenshot):
```
Verifying integrity of required files...
All required files are present and verified.

Killing Chrome and Edge processes...

Searching for infected scripts...
The following files are potentially infected:
C:\Windows\System32\NvWinSearchOptimizer.ps1
Write-Host : Cannot bind parameter 'ForegroundColor'. Cannot convert value "WARNING" to type "System.ConsoleColor".
Error: "Unable to match the identifier name WARNING to a valid enumerator name. Specify one of the following
enumerator names and try again:
Black, DarkBlue, DarkGreen, DarkCyan, DarkRed, DarkMagenta, DarkYellow, Gray, DarkGray, Blue, Green, Cyan, Red,
Magenta, Yellow, White"
At C:\Users\Cian\Desktop\yoursearchbar.me-removal-master\removal.ps1:117 char:72
+ ... -Host "`nDo you want to remove them? [Y/N]"  -ForegroundColor WARNING
+                                                                   ~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Write-Host], ParameterBindingException
    + FullyQualifiedErrorId : CannotConvertArgumentNoMessage,Microsoft.PowerShell.Commands.WriteHostCommand

```

![image](https://github.com/E-B3rry/yoursearchbar.me-removal/assets/49107558/af32461a-3f33-4f0c-958f-8d224f8a5610)

**Comment by Cian-R** [comment]
Follow-up: deleting ```C:\Windows\System32\NvWinSearchOptimizer.ps1``` did not fix the issue.

**Comment by E-B3rry** [comment]
Thank you for opening an issue, I fixed the initial issue. However, the script should have done more than just deleting the file `C:\Windows\System32\NvWinSearchOptimizer.ps1` to remove the malware from you computer. The extension is still here and persisting after the script ran? And if so, did the deleted PowerShell file came back? If so, could you please join it here or send it in private so I can see if you have a different version? I still wasn't able to get my hand of the latest version.

**Comment by Cian-R** [comment]
Apologies, I should have followed this up. After deleting NVWinSearchOptimiser.ps1 I saw that the issue still existed so I made the follow-up comment on this issue. A little while after that I ran your program again, and it fully completed. This time it worked and the malware is no longer affecting my browser which is fantastic, thank you.
Side affects included:
-Edge stopped working, but a quick repair seems to have fixed it and it works fine again.
-The new outlook app for desktop no longer works and I have not yet found a fix (no issue, I can still use their webapp)
